jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -143,11 +144,13 @@ extern ecma_value_t ecma_make_object_value (const ecma_object_t *);
 extern ecma_value_t ecma_make_error_value (ecma_value_t);
 extern ecma_value_t ecma_make_error_obj_value (const ecma_object_t *);
 extern ecma_integer_value_t ecma_get_integer_from_value (ecma_value_t);
+extern ecma_number_t ecma_get_float_from_value (ecma_value_t value);
```

Comment:
These new functions were tagged with `__attr_pure___` at the definition, shouldn't it be added here as well?